15.0/apache/Dockerfile

```diff
@@ -133,19 +132,23 @@ RUN set -ex; \
 # gpg key from https://nextcloud.com/nextcloud.asc
     gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys 28806A878AE423A28372792ED75899B9A724937A; \
     gpg --batch --verify nextcloud.tar.bz2.asc nextcloud.tar.bz2; \
-    tar -xjf nextcloud.tar.bz2 -C /usr/src/; \
+    tar -xjf nextcloud.tar.bz2 -C /var/www/; \
+    rm -rf /var/www/html; \
+    mv /var/www/nextcloud /var/www/html; \
     gpgconf --kill all; \
-    rm -r "$GNUPGHOME" nextcloud.tar.bz2.asc nextcloud.tar.bz2; \
-    rm -rf /usr/src/nextcloud/updater; \
-    mkdir -p /usr/src/nextcloud/data; \
-    mkdir -p /usr/src/nextcloud/custom_apps; \
-    chmod +x /usr/src/nextcloud/occ; \
+    rm -r "$GNUPGHOME"; \
+    rm -rf /var/www/html/updater; \
+    mkdir -p /var/www/html/data; \
+    mkdir -p /var/www/html/custom_apps; \
+    chmod +x /var/www/html/occ; \
+    touch /var/www/html/data/.ocdata; \
+    chown -R www-data:root /var/www/html
```

Comment:
Seems like this should be `chown -R www-data:root /var/www/html; \`. Currently docker reports error: `Dockerfile parse error line 146: unknown instruction: APT-GET`